src/test/store/custom-assertions.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+export function assertSetContainsOnly<T>(set: Iterable<T>, expected: Array<T>) {
```

Comment:
Potential follow-up: Should this go in some kind of utils folder? I think the fixtures folder is where we have been putting them. I think it may be good to re-organize things. I filed #862, feel free to comment there if you have opinions.

Reply:
I don't have strong opinions. I added it here more because it's used in this directory, but it's totally true that it can be used in a lot of other places too. I'm fine moving it when we need it.

Reply:
Or now :)

Reply:
I moved it to `fixtures`